tutorial/westeros/westeros_addon_technologies.ipynb

```diff
@@ -0,0 +1,607 @@
+    "\n",
+    "District heat (`dh_grid`) network characteristics: District heating networks have only very low losses as these cover only short distances (within city perimeters). We will assume the district heating network to have an efficiency of 97%."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As before we will work our way backwards, starting from the `heat` demand defined at the `useful` energy level and connecting this to the `final` energy level via a technology, `dh_house`, representing the in-house heat distribution system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tec = 'dh_house'\n",
+    "scen.add_set('technology', tec)\n",
+    "\n",
+    "dh_house_out = make_df(base_output, technology=tec, commodity='heat', \n",
+    "                   level='useful', value=1.0)\n",
+    "scen.add_par('output', dh_house_out)\n",
+    "\n",
+    "dh_house_in = make_df(base_input, technology=tec, commodity='heat',  \n",
+    "                  level='final', value=1.0, unit='%')\n",
+    "scen.add_par('input', dh_house_in)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we will add the district heating network"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tec = 'dh_grid'\n",
+    "scen.add_set('technology', tec)\n",
+    "\n",
+    "dh_grid_out = make_df(base_output, technology='dh_grid', commodity='heat', \n",
+    "                   level='final', value=1.0)\n",
+    "scen.add_par('output', dh_grid_out)\n",
+    "\n",
+    "dh_grid_in = make_df(base_input, technology='dh_grid', commodity='heat',  \n",
+    "                  level='secondary', value=1.03, unit='%')\n",
+    "scen.add_par('input', dh_grid_in)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The last technology we will add is the `po_turbine` itself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tec = 'po_turbine'\n",
+    "scen.add_set('technology', tec)\n",
+    "\n",
+    "po_out = make_df(base_output, technology=tec, commodity='heat', \n",
+    "                   level='secondary', value=1.0)\n",
+    "scen.add_par('output', po_out)\n",
+    "\n",
+    "po_in = make_df(base_input, technology=tec, commodity='electricity',  \n",
+    "                  level='secondary', value=0.2, unit='%')\n",
```

Comment:
Please make sure there is explanation for using this number 0.2.

Reply:
it was actually explained above, but i detailed this some more as it wasnt clear enough.